src/components/SingleStoryCard/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Row, Col, Image } from "react-bootstrap";
+import { NavLink } from "react-router-dom";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const SingleStoryCard = ({ post, order }) => {
+  const { t } = useTranslation();
+
+  return (
+    <Row className="justify-content-md-center articleContainer">
+      <Col lg={{ span: 3, order: order }} className="articleImage">
+        <Image
+          src={post.jetpack_featured_media_url}
+          className="roundImage"
+          roundedCircle
+        />
+      </Col>
+      <Col lg="6" className="blogCardBox">
+        <h3
+          className="articleTitle"
+          dangerouslySetInnerHTML={{
+            __html: post.title.rendered,
+          }}
+        ></h3>
+        <span
+          className="articleBody"
+          dangerouslySetInnerHTML={{
+            __html: post.excerpt.rendered,
+          }}
+        ></span>
+        <NavLink to={`/blog/${post.id}`}>
```

Comment:
now here's whats confusing me all the blogs are in a url which is /stories and the single story is in /blog/:id so please either change the stories components to be blogs or change the blog to be story, the same thing with the components name

Reply:
and merge it with the latest update of dev branch

Reply:
Resolved